README.md

```diff
@@ -1,122 +1,58 @@
 # Workshops
 
+This is the home base of ClojureBridge workshops.
+
+---
+
+## Contents
+
+### [:rocket: Getting Started](#rocket-getting-started)
+### [:memo: Guide to listing your event](./docs/workshop-listing-guide.md)
```

Comment:
I actually prefer sentence case, but we use title case elsewhere so we should do so here too:

```suggestion
### [:memo: Guide to Listing Your Event](./docs/workshop-listing-guide.md)
```